### PR TITLE
dev-python/polib: version bump

### PR DESCRIPTION
**Pull request by nE0sIghT**
1. EAPI=6
2. Changed SRC_URI

### CODE REVIEW

File: dev-python/polib/polib-1.0.7.ebuild [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+		emake html
+	fi
+}
+
+python_test() {
+	python tests/tests.py || die
+}
+
+python_install_all() {
+	local DOCS="CHANGELOG README.rst"
```
Comment (rindeal) [inline dev-python/polib/polib-1.0.7.ebuild]: please use an array

File: dev-python/polib/polib-1.0.7.ebuild [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+python_compile_all() {
+	if use doc; then
+		cd docs || die
+		emake html
+	fi
+}
+
+python_test() {
+	python tests/tests.py || die
```
Comment (rindeal) [inline dev-python/polib/polib-1.0.7.ebuild]: shouldn't this be `${PYTHON} ...`?

**Comment by nE0sIghT** [comment]
Pushed fixes

File: dev-python/polib/polib-1.0.7.ebuild (hunk 1 above)
Comment (idella) [inline dev-python/polib/polib-1.0.7.ebuild]: both work

Comment (nE0sIghT) [inline dev-python/polib/polib-1.0.7.ebuild]: It's documented as array: https://github.com/gentoo/gentoo/blob/master/eclass/distutils-r1.eclass#L124

**Comment by idella** [comment]
1. doc? ( dev-python/sphinx ) requires  dev-python/sphinx[${PYTHON_USEDEP}]

2. `python tests/tests.py || die`   calls the system python which with 3 impls of python set in 
PYTHON_COMPAT results in onlt the one impl run 3 times

3. can you bump EAPI to 6
4. also add support to py3.5

**Comment by idella** [comment]
On substituting EAPI to 6 and adding py3.5 support, all passes all

An  alternative to the test phase is simply

`./runtests.sh`

which will run each required python impl.

**Comment by nE0sIghT** [comment]
1-2,4-5 done
EAPI already at version 6

**Comment by idella** [comment]
ok I shall edit the remainder on commit since they are trivial.  They are carry over from the initial commit.  I shall outline them here so as to inform:

```ebuild
RDEPEND=""
DEPEND="${RDEPEND}
        doc? ( dev-python/sphinx[${PYTHON_USEDEP}] )
"
```

In recent EAPIs, if some vars are empty, they can be left out. This pertains to RDEPEND=""
DEPEND="${RDEPEND} os therefore un-needed, leaving

```ebuild
DEPEND="doc? ( dev-python/sphinx[${PYTHON_USEDEP}] )"
```

${PYTHON} requires " ", which you seemingly mistranscribed

**Comment by nE0sIghT** [comment]
Pushed fixes

**Comment by idella** [comment]
ok I shall edit the remainder on commit since they are trivial.  They are carry over from the initial commit.  I shall outline them here so as to inform:

```ebuild
RDEPEND=""
DEPEND="${RDEPEND}
        doc? ( dev-python/sphinx[${PYTHON_USEDEP}] )
"
```

In recent EAPIs, if some vars are empty, they can be left out. This pertains to RDEPEND=""
DEPEND="${RDEPEND} os therefore un-needed, leaving

```ebuild
DEPEND="doc? ( dev-python/sphinx[${PYTHON_USEDEP}] )"
```

${PYTHON} requires " ", which you seemingly mistranscribed

**Comment by idella** [comment]
Oh I forgot to remove 
local DOCS=( CHANGELOG README.rst )
(next time)

**Pull request closed without merging**